test/integration/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ set(test_name integration)
 
 include(FindPythonInterp)
 
-add_test(NAME ${test_name}
+if(USE_PMEM)
```

Comment:
nit:
I think this is else could be simplified:
if(USE_PMEM)
    set(test_arg --pmem)
endif()
and use
COMMAND ${PYTHON_EXECUTABLE} test_twemcache.py ${test_arg}